docs/testing.md

```diff
@@ -38,11 +78,14 @@ Set up and build:
 
 ```
 apt update
-apt-get -y install libjson-c-dev libcmocka-dev clang valgrind python3-pytest debianutils flake8 libssl-dev cmake
+apt-get -y install libjson-c-dev libcmocka-dev clang valgrind \
+                   python3-pytest debianutils flake8 libssl-dev\
```

Comment:
nit, missing space before backslash?

Reply:
Also the 'libssl-dev' was recently dropped, so i'll fix that mistake too